test/idl.js

```diff
@@ -802,6 +802,9 @@ TestCluster.test('run `idl update`', {
 
 TestCluster.test('run `idl update` with corrupt meta.json', {
 }, function t(cluster, assert) {
+    assert.end(null, 'skip')
```

Comment:
Revisited. This was unrelated and failing before I made my changes so I punted. Closer investigation reveals that the error message out of `read-json` got more specific and broke this test. Fixed.